src/modules/toggle_completed.js

```diff
@@ -0,0 +1,19 @@
+// Update completed status
+const toggleCompleted = (toDoList, taskManager, index) => {
+  const toDoItem = toDoList.querySelector(`[data-index="${index}"]`);
+  if (!toDoItem) {
+    return;
+  }
+  const checkButton = toDoItem.querySelector('.check-button');
```

Comment:
- [ ] Good job so far 👏 But when I clicked on the check button the status is not updated (completed: true/false) as you can see on the screenshot below ⬇️. Please make sure that your event listener to the checkbox (change) is well implemented and fix it in order to make your app works as expected 👍 
![checkkk](https://user-images.githubusercontent.com/6615394/236452326-02ca2520-5fc8-4f33-9b1f-362c189739fb.PNG)